src/components/CustomForm.tsx

```diff
@@ -1,51 +1,46 @@
-
 // library imports
 import { AiFillPlusSquare } from "solid-icons/ai";
-import { createSignal } from 'solid-js';
+import { createSignal } from "solid-js";
+
+interface Props {
```

Comment:
I know it seems tedious at first, but I would name your prop types with the component name in the title, e.g. CustomFormProps. That way if you need them outside of here, you can easily get auto-imports for them and they won't pollute your import namespace.